### PR TITLE
Standardize attributes for weights and objectives

### PR DESCRIPTION
**Pull request by ajreynol**

### CODE REVIEW

File: language/sygus.tex [hunk 1]
```diff
@@ -1905,7 +1905,8 @@ \subsubsection{Weights}
 k_{def} & \text{otherwise, where } k_{def} \text{ is the default weight value for } \texttt{W}
 \end{cases}
 \]
-Unless stated otherwise, the default weight value for all weight keywords is $0$.
+The default weight value for all weight keywords is $0$.
+This value can be overidden via an attribute as described in \cref{sec:attr-weights}.
```
Comment (SaswatPadhi) [inline language/sygus.tex]:
```suggestion
This value can be overridden via an attribute as described in \cref{sec:attr-weights}.
```

File: language/sygus.tex [hunk 2]
```diff
@@ -2402,12 +2403,73 @@ \subsection{Oracle Example}
...
+\end{enumerate}
+
+\paragraph{Tuple Orderings}
+
+\begin{enumerate}
+\item The attribute $\texttt{:lexico}$ specifies that the ordering on tuples is lexicographic.
+In particular, assuming orderings $\succ_1, \ldots, \succ_n$ on the components of $n$-tuples,
+$(c_1, \ldots, c_n) \succ (d_1, \ldots, d_n)$ holds if there exists a $0 \leq j \leq n$
+such that $c_i = d_i$ for $i<j$ and $c_j \succ_j d_j$.
+This attribute expects no provided attribute value.
```
Comment (SaswatPadhi) [inline language/sygus.tex:2462]: One question here is, what should be the behavior when a value is provided after this attribute? Would this be a syntax error, or would it be ignored?

File: language/sygus.tex [hunk 3]
```diff
@@ -2683,7 +2745,9 @@ \section{Examples}%
 \end{lstlisting}
 In this example, a weight keyword $\texttt{numX}$ has been declared.
 A function-to-synthesize $f$ has been declared, whose production
-rules include annotations that count the number occurrences of $x$.
+rules include annotations.
+These annotations count the number occurrences of $x$, where recall
+that the default value for unannotated terms with repsect to the given weight is $0$.
```
Comment (SaswatPadhi) [inline language/sygus.tex]:
```suggestion
that the default value for unannotated terms with respect to the given weight is $0$.
```

**Review by SaswatPadhi — APPROVED** [review]
Some minor comments below:

File: language/sygus.tex (hunk 2 above)
Comment (ajreynol) [reply language/sygus.tex:2462]: This would be a syntax error.

Comment (SaswatPadhi) [reply language/sygus.tex:2462]: Thanks Andy. Should we add a sentence about this in the syntax section may be?

I've also approved this PR, so please feel free to merge whenever you're happy with it.

Comment (ajreynol) [reply language/sygus.tex:2462]: My intention was to make this section the reference on the concrete syntax, so this sentence should be taken as the authority.

Comment (SaswatPadhi) [reply language/sygus.tex:2462]: I see, that makes sense too. Thanks.

**Pull request merged**